src/reducers/ticketReducers.js

```diff
@@ -45,6 +45,18 @@ const addIncidentsToState = (state, incidents) => {
   return newState
 }
 
+const removeTicketFromState = (state, ticketId) => {
```

Comment:
The kids these days seem to do it differently, but back in the day we would have put these at the bottom of the file as function expressions. I think what I really want is a separation between things that are public (exported) and things that are not for readability.

Reply:
I moved all the non-public items to the top.  They needed to be declared before they were used.